app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -243,7 +243,8 @@ def push_manifest
     end
 
     def pulp_content
-      Katello.pulp_server.resources.content
+      # TODO: Remove this
+      fail 'Pulp 2 content was being used'
```

Comment:
How to proceed with this? I don't know enough about this controller to say how to replace this.

Reply:
This code relates to tagging manifests and uploading manifests via Pulp 2. We already have a different code path for tagging manifests with Pulp 3 (https://github.com/Katello/katello/pull/9054/files), but no way to upload manifests. I actually already have an issue for this: https://projects.theforeman.org/issues/33767

For the sake of this PR, I'd say you can rip all the code out relating to uploading manifests. So that would be `pulp_content`, `upload_manifest`, and `upload_tag`. That also means we'll need to remove mention of uploading docker manifests from other bits of the code:

- https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/import_upload.rb#L82-L91
- Some of the controllers mention uploading manifests. Those references can be removed.
- https://github.com/Katello/katello/blob/master/lib/katello/repository_types/docker.rb#L30

Reply:
Note that we do still support removing manifests.